lib/oli_web/controllers/pow/session_html/new.html.heex

```diff
@@ -1,62 +1,114 @@
 <%= if OliWeb.Pow.PowHelpers.current_pow_config(@conn) == Oli.Accounts.Author do %>
-  <%= render OliWeb.SharedView, "_box_form_container.html", Map.merge(assigns, %{title: value_or(assigns[:title], "Authoring Sign In"), bs_col_class: "mx-auto mt-10"}) do %>
-    <%= for link <- OliWeb.Pow.PowHelpers.provider_links(@conn), do: raw(link) %>
-    <hr class="my-4" />
-
-    <%= form_for @changeset, @action, [as: :user], fn f -> %>
-      <div class="form-label-group">
-        <%= email_input(f, Pow.Ecto.Schema.user_id_field(@changeset),
-          class: "form-control",
-          placeholder: "Email",
-          required: true,
-          autofocus: true
-        ) %>
-        <%= label(f, Pow.Ecto.Schema.user_id_field(@changeset), class: "control-label") %>
-        <%= error_tag(f, Pow.Ecto.Schema.user_id_field(@changeset)) %>
-      </div>
-      <div class="form-label-group">
-        <%= password_input(f, :password,
-          class: "form-control",
-          placeholder: "Password",
-          required: true
-        ) %>
-        <%= label(f, :password, class: "control-label") %>
-        <%= error_tag(f, :password) %>
+  <div class="relative h-[calc(100vh-180px)] flex justify-center items-center">
+    <div class="absolute h-[calc(100vh-180px)] w-full top-0 left-0">
+      <.author_sign_in />
+    </div>
+    <div class="flex flex-col gap-y-10 lg:flex-row w-full relative z-50 overflow-y-scroll lg:overflow-y-auto h-[calc(100vh-270px)] md:h-[calc(100vh-220px)] lg:h-auto py-4 sm:py-8 lg:py-0">
+      <div class="w-full lg:w-1/2 flex items-start lg:pt-10 justify-center">
+        <div class="w-96 flex-col justify-start items-start gap-3.5 inline-flex">
+          <div class="text-center">
+            <span class="text-white text-4xl font-normal font-['Open Sans'] leading-10">
+              Welcome to
+            </span>
+            <span class="text-white text-4xl font-bold font-['Open Sans'] leading-10">OLI</span><span class="text-white text-4xl font-normal font-['Open Sans'] leading-10"> Torus</span>
```

Comment:
We should not embed "OLI" or "Torus" directly into the codebase as the name of the product.  

Instead, use the values from the appropriate Vendor Property.  In `config.exs` you can see the collection of them and their default values:

```
config :oli, :vendor_property,
  workspace_logo:
    System.get_env("VENDOR_PROPERTY_WORKSPACE_LOGO", "/branding/prod/oli_torus_icon.png"),
  product_full_name:
    System.get_env("VENDOR_PROPERTY_PRODUCT_FULL_NAME", "Open Learning Initiative"),
  product_short_name: System.get_env("VENDOR_PROPERTY_PRODUCT_SHORT_NAME", "OLI Torus"),
  product_description:
    System.get_env(
      "VENDOR_PROPERTY_PRODUCT_DESCRIPTION",
      default_description
    ),
  product_learn_more_link:
    System.get_env("VENDOR_PROPERTY_PRODUCT_LEARN_MORE_LINK", "https://oli.cmu.edu"),
  company_name: System.get_env("VENDOR_PROPERTY_COMPANY_NAME", "Carnegie Mellon University"),
  company_address:
    System.get_env(
      "VENDOR_PROPERTY_COMPANY_ADDRESS",
      "5000 Forbes Ave, Pittsburgh, PA 15213 US"
    ),
  support_email: System.get_env("VENDOR_PROPERTY_SUPPORT_EMAIL")
```

Reply:
Okey, I understand. I just need to make some modifications here because if I use `product_short_name` it will correctly bring me the text I need to display but I need to apply different styles to OLI and Torus, as the word OLI should be bold and Torus should not. 

Both words have different styles to be applied when rendered in this view according to the design I'm following (and in the other login views for students and instructors).